tests/unit/combine_results/combined_MotEvo_results.tsv

```diff
@@ -0,0 +1,22 @@
+strand	binding_position	binding_posterior	pwm_id	binding_region	binding_sequence	binding_energy	fasta_record

```

Comment:
I would rearrange the order of columns here a little:
`pwm_id binding_position, binding_sequence	binding_posterior	binding_energy`

We do not need `strand` or `fasta_record` for now. Please keep them in the code but just do not use them to generate the table. The same applies to `binding_region` - note that it is the same as `fasta_record`

Reply:
Done!